packages/core/stories/file-drop-zone/file-drop-zone.qa.stories.tsx

```diff
@@ -2,12 +2,12 @@ import {
   FileDropZone,
   FileDropZoneIcon,
   FileDropZoneTrigger,
-} from "@salt-ds/lab";
+} from "@salt-ds/core";
 import { Meta, StoryFn } from "@storybook/react";
 import { AllRenderer } from "docs/components";
 
 export default {
-  title: "Lab/File Drop Zone/QA",
+  title: "Core/File Drop Zone/QA",
```

Comment:
```suggestion
  title: "Core/File Drop Zone/File Drop Zone QA",
```
for easier recognition on chromatic :D